meshnesh/android-freckle
# Ch create readmedoc 163479506

### what should this PR do?
Integrate the project with firebase crashlytics and add the README doc

#### Description of tasks to be completed?
Integrate application to firebase crashlytics tracking

### How can this be manually tested?
* check the main repo page to see the README doc.
* get access to the firebase console to check the crashlytics report

### Relevant PT stories
* [Integrate Performance Monitoring](https://www.pivotaltracker.com/story/show/163186994)
[Create README doc](https://www.pivotaltracker.com/story/show/163479506)

### Screenshots
![screenshot 2019-01-29 at 13 28 56](https://user-images.githubusercontent.com/15923109/51976945-1e635a00-2497-11e9-93f5-8fa659bac68a.png)
